docs/README.md

```diff
@@ -0,0 +1,122 @@
+# Developer Documentation for Compensated
+
+Compensated intends to make it safe and easy for people to begin taking payments
+from their customers directly within their application from a variety of payment
+sources. Compensated is packaged and distributed as a Ruby library ([source
```

Comment:
By just reading this statement, I had an impression of this library will handle the full cycle of payment processing where I think an architecture diagram or explicitly saying the current state of this project would be useful.

Reply:
Yea, I think we want to move toward that degree of functionality. I wonder if we could ask a few of the folks who have reached out to us what they need the most help with on the "front-end" side; as the current design was optimized for systems where the "front" of the funnel was handled by TypeForm, iOS, and Gumroad respectively.